app/boards/arm/corneish_zen/corneish_zen_v2_left_defconfig

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2022 Darryl deHaan
+# SPDX-License-Identifier: MIT
+#
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_CORNEISH_ZEN_V2_LEFT=y
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SLEEP=y
```

Comment:
Should deep sleep be enabled by default? I don't think any other board/shield sets that in ZMK at the moment.

Reply:
We don't have a firm policy here. If a specific *vendor* board wants to enable this, I don't see a problem with it.

Thoughts?

Reply:
I am in general slightly opposed, since it can cause confusion to the users if they don't know that deep sleep is enabled. If they have to enable it in the conf themselves they will at least be aware of it. However I do see the logic in letting the vendor/designer decide for their own keyboards.

Reply:
My main concern was folks who are firmware unaware. At least they would get the battery savings.